src/EpisodeDataService/GetEpisode/GetEpisode.cs

```diff
@@ -22,15 +22,20 @@ public GetEpisode(ILogger<GetEpisode> logger, IEpisodeRepository episodeReposito
     [Function("GetEpisode")]
     public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequestData req)
     {
-        string episodeId;
+        string episodeIdString = req.Query["episodeId"];
 
-        episodeId = req.Query["episodeId"];
-        if (string.IsNullOrEmpty(episodeId))
+        if (string.IsNullOrEmpty(episodeIdString))
         {
             _logger.LogError("Episode ID is not provided.");
             return req.CreateResponse(HttpStatusCode.BadRequest);
         }
 
+        if (!long.TryParse(episodeIdString, out long episodeId))
+        {
+            _logger.LogError("Invalid episode ID format.");
+            return req.CreateResponse(HttpStatusCode.BadRequest);
+        }
```

Comment:
You can remove the first if statement since the second if statement will also catch null or empty values